src/test/java/com/diffmin/MainTest.java

```diff
@@ -0,0 +1,79 @@
+package com.diffmin;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.FileNotFoundException;
+import java.security.Permission;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/** Unit test to verify Main method's functioning */
+public class MainTest {
+    MainTest() {
+        System.setSecurityManager(new DoNotExitJVM());
+    }
+
+    @SuppressWarnings("serial")
```

Comment:
Most places they are recommending to explicitly mention `serialVersionUID` and I understood the reason but I did not get the intuition of serializing and deserializing. Let me go through the concept and I will fix this.

Reply:
I wouldn't bother in a test suite, to be honest. We have no intention of serializing that exception.

Reply:
It doesn't really matter. I wouldn't bother putting it in, and similarly wouldn't bother taking it out :)